csrc/utils.h

```diff
@@ -551,4 +551,17 @@ V getOrDefault(const std::unordered_map<K, V>& map, const K& key) {
   return i == map.end() ? V() : i->second;
 }
 
+inline int64_t wrapDim(int64_t dim, int64_t ndim) {
```

Comment:
Note on non-int64 changes. We had be doing this manually many times, so I created a utility function for it.